File: 326 Power of Three.kt
class Solution {
    fun isPowerOfThree(n: Int): Boolean 
	{
        return (n > 0 && 1162261467 % n == 0);      // 3 ^ 19 = 1162261467

	}    
}

// Status: Accepted
// Runtime: 248 ms
// Memory Usage: 35.5 MB

File: 326 Power of Three.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfThree = function(n) {
    if (n === 1) 
        return true
    if ( n < 1) 
        return false
    return isPowerOfThree(n/3)
};

// 04/28/2021 14:17	Accepted	236 ms	49.1 MB	javascript
